src/components/RichTextEditor/serializer/markdown/types.ts

```diff
@@ -81,7 +81,7 @@ export type BlockType = {
     children: NodeType[];
     parentType?: string;
     caption?: string;
-    language?: string;
+    language?: string | null;
```

Comment:
We need to decide between `undefined` or `null` :p

Reply:
Updated

Reply:
The null i get from passer could only transform to undefined :)